### ISSUE TITLE
When a web dialog is opened, NVDA does not read the dialog title and only reads the focused control

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13507]
### Steps to reproduce:
Open this URL in Edge/Chrome: https://codepen.io/hoovercj/pen/eYRNjbB
Navigate to the "Show TeachingBubble" button 
Press Enter to activate

### Actual behavior:
A dialog/popup appears. NVDA reads the currently focused control ("maybe later") but does not read the dialog title ("discover what's trending around you")

### Expected behavior:
NVDA should read the dialog title AND the focused control

#### NVDA installed/portable/running from source:
Repros with installed AND portable
This issue does not occur with Narrator or JAWS.

#### NVDA version:
2021.3.4

#### Windows version:
(OS Build 22458.1000)

#### Name and version of other software in use when reproducing the issue:
Edge, version 93.0.961.52

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2021.1 also repros this issue

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Brian1Gaff** [comment]
I think its not just web dialogues and often the navigation mode needs to be 
changed to manually read some of the message boxes, so the data is there, 
its just not being flagged to nvda that it should be reading it.

Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "msftedad" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Friday, March 18, 2022 8:33 PM
Subject: [nvaccess/nvda] When a web dialog is opened, NVDA does not read the 
dialog title and only reads the focused control (Issue #13507)

**Comment by michaelDCurran** [comment]
The heading node  needs to be associated with the dialog using aria-labelledby. This is not an issue with NVDA.

**Comment by XLTechie** [comment]
I'm curious: how did you get to the default issue template, instead of the 
preferred bug reporting template reached by going to
https://github.com/nvaccess/nvda/issues/
and pressing the "new issue" button, which allows you to choose between bug or 
feature report?

Was this one linked from somewhere?

**Comment by seanbudd** [comment]
@XLTechie some GitHub UX paths lead to https://github.com/nvaccess/nvda/issues/new
